README.md

```diff
@@ -103,3 +103,8 @@ Below is an example of regexp patterns used for autolinking, modified in the `co
 3. Using autolinking to create group mentions. Use:
   - Pattern: `@customgroup*`
   - Template: `[@customgroup]( \\* @user1 @user2 @user3 \\* )`
+  
+4. For servers with multiple domains (like community and community-daily on the offical Mattermost Server) a substition of absolute conversation links to relative links is recommended to prevent issues in the mobile app. Add one pattern for each domain Used:
+  - Pattern: `https://community\\.mattermost\\.com/core/pl/(?P\u003cid\u003e[a-zA-Z0-9]+)`
+  - Template: `[\u003cjump to convo\u003e](/core/pl/${id})`
```

Comment:
Maybe you can also replace the team name? This would make it more independent and a better Copy&Past example.

Reply:
Sure I could do that. What would you recommend? Something like:
``https://subdomain\\.domain\\.tld/teamname/pl/(?P\u003cid\u003e[a-zA-Z0-9]+)``

Reply:
I would like to capture the team name as part of a group.

`https://subdomain\\.domain\\.tld/(?P\u003cteamname\u003e(?a-zA-Z0-9]+)/pl/(?P\u003cid\u003e[a-zA-Z0-9]+)`

`[\u003cjump to convo\u003e](/${teamname}/pl/${id})`

Site note: Why are you using `\u003c` instead of `<`?

Reply:
@svelle What do you think about this?

Reply:
Oh this went under my radar, sorry. I pretty much copy pasted the command I got from carlos for this and this is what was working on my testinstance so I didn't bother changing it afterwards.

Same for the teamname. I'm open to changing it though